samples/ar-view-node/src/main/java/io/github/sceneview/sample/arviewnode/utils/NodeRotationHelper.kt

```diff
@@ -0,0 +1,68 @@
+package io.github.sceneview.sample.arviewnode.utils
+
+import com.google.ar.core.TrackingState
+import io.github.sceneview.ar.ARSceneView
+import io.github.sceneview.ar.node.AnchorNode
+import io.github.sceneview.collision.Quaternion
+import io.github.sceneview.collision.Vector3
+import io.github.sceneview.math.toVector3
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+class NodeRotationHelper {
+
+    private var skipCounter = 0
+    private var isRunning = false
+
+    fun updateRotationOfViewNodes(
+        sceneView: ARSceneView,
+        nodeList: MutableList<AnchorNode>
+    ) {
+        if (isRunning) {
+            return
+        }
+
+        if (skipCounter <= MAX_SKIP_FRAMES) {
+            skipCounter++
+            return
+        }
+
+        skipCounter = 0
+
+        GlobalScope.launch {
+            withContext(Dispatchers.IO) {
```

Comment:
I still have doubts about this since it can cause issues if people copy this approach.

Reply:
I removed it and added some docstrings.